Docs/2015.md

```diff
@@ -0,0 +1,121 @@
+#iWork '15+ File Format
+
+## <a name="overview"/>Overview
+The iWork '15+ format is a [bundle](https://developer.apple.com/library/mac/documentation/corefoundation/conceptual/cfbundles/DocumentPackages/DocumentPackages.html)-based format built on top of several open source projects. This document describes the physical layout of files contained in these bundles and the algorithms involved, but does not attempt to describe the nature of the represented object graph.
+
+## <a name="work"/>What does work
+
+It works in the same way of the old project, but now you can choose also to open the new file format (2015+)
+
+## <a name="no-work"/>What doesn't
+
+Documents protected with password, we need to investigate about this because something is changed in the new project format (there is no .iwph file)
```

Comment:
I created a password protected presentation using Keynote 7.0.5 (iWork 2016 update) and both `.iwph` + `iwpv2` files are present in the `.key` file.